vmm/src/cpu.rs

```diff
@@ -141,10 +155,14 @@ pub enum Error {
     #[error("Error during CPU debug: {0}")]
     CpuDebug(#[source] hypervisor::HypervisorCpuError),
 
-    #[cfg(all(target_arch = "x86_64", feature = "gdb"))]
+    #[cfg(all(feature = "gdb", target_arch = "x86_64"))]
```

Comment:
Nit: `feature` and `feature_arch` are reversed compared to other places in the code.